notebooks/stac-load-e84-aws.py

```diff
@@ -78,10 +78,10 @@ def convert_bounds(bbox, invert_y=False):
 r = 100 * km2deg
 bbox = (x - r, y - r, x + r, y + r)
 
-catalog = Client.open("https://earth-search.aws.element84.com/v0")
+catalog = Client.open("https://earth-search.aws.element84.com/v1/")
 
 query = catalog.search(
-    collections=["sentinel-s2-l2a-cogs"], datetime="2021-09-16", limit=100, bbox=bbox
+    collections=["sentinel-2-l2a"], datetime="2020-01-01/2020-02-31", limit=100, bbox=bbox
```

Comment:
Why was change in date necessary? This one queries 2 months worth of data, a bit too much for github pipelines.

Reply:
Thanks @Kirill888 Fixed date, removed config, and made a few more changes. Ran the code locally and it runs all fine.